### PR TITLE
Migrate the entire `lib/` directory to null-safety.

### PR DESCRIPTION
**Pull request by pennzht**
This PR migrates the `lib/` directory of `google_fonts` to null-safety, allowing the package to be published as null-safe.

The `generator/` and `test/` directories are not fully migrated yet.

Temporarily switches to flutter `master` to resolve version conflict between
* flutter_test (depending on typed_data 1.3.0-nullsafety.5)
* crypto (depending on typed_data ^1.3.0)

Plan to switch back to `beta` once it uses typed_data 1.3.0

### CODE REVIEW

**Review by JoseAlba — APPROVED** [review]
```
██╗      ██████╗ ████████╗███╗   ███╗
██║     ██╔════╝ ╚══██╔══╝████╗ ████║
██║     ██║  ███╗   ██║   ██╔████╔██║
██║     ██║   ██║   ██║   ██║╚██╔╝██║
███████╗╚██████╔╝   ██║   ██║ ╚═╝ ██║
╚══════╝ ╚═════╝    ╚═╝   ╚═╝     ╚═╝
```

**Review by rami-a — APPROVED** [review]

**Review by clocksmith — APPROVED** [review]

File: pubspec.yaml [hunk 1]
```diff
@@ -18,16 +18,10 @@ dependencies:
```
Comment (kevmoo) [inline pubspec.yaml:18]: All of the comments above ⬆️  can be removed now, right?

Comment (pennzht) [reply pubspec.yaml]: Nice catch, thank you!
Fixed.

File: .github/workflows/actions.yml [hunk 2]
```diff
@@ -17,7 +17,11 @@ jobs:
         uses: actions/checkout@v2
         with:
           repository: 'flutter/flutter'
-          ref: 'beta'
+          ref: 'master'
```
Comment (kevmoo) [inline .github/workflows/actions.yml:20]: move this back to `beta`!!! Need to validate there

Comment (kevmoo) [reply .github/workflows/actions.yml]: 👍

**Comment by kevmoo** [comment]
See https://github.com/material-foundation/google-fonts-flutter/pull/144

This includes rebasing on develop and getting tests to be null safe

**Comment by kevmoo** [comment]
@clocksmith – we can close this out then!

**Pull request closed without merging**